source/iterators.tex

```diff
@@ -4945,7 +4969,8 @@
       requires Assignable<I&, const I2&>
         constexpr counted_iterator& operator=(const counted_iterator<I2>& x);
 
-    constexpr I base() const;
+    constexpr I base() const & requires CopyConstructible<I>;;
```

Comment:
Two semi-colons at the end of this line.

Reply:
thanks!